Makefile

```diff
@@ -86,3 +89,18 @@ docker-run:
 
 docker-run-db:
 	@docker-compose up db
+
+pre-build:
+	@rm -rf .build || true
+	@mkdir -p .build;
+	@rm -rf terraform/.artifacts || true
+	@mkdir -p terraform/.artifacts
+
+api-build: pre-build
+	@yarn install
+	@yarn workspace @symchk/api build
+	@yarn workspaces focus @symchk/api --production
```

Comment:
Does this clean up the contents of `node_modules` folder from the yarn install two lines above?

Reply:
Yes it will. We need the install for the build to work